sdk/storage/azure-storage-blob/azure/storage/blob/aio/_container_client_async.py

```diff
@@ -872,38 +872,18 @@ async def delete_blobs(  # pylint: disable=arguments-differ
         Soft deleted blobs or snapshots are accessible through :func:`list_blobs()` specifying `include=["deleted"]`
         Soft-deleted blobs or snapshots can be restored using :func:`~BlobClient.undelete()`
 
-        :param blobs: The blob names with which to interact. This can be a single blob, or multiple values can
+        :param blobs:
+            The blobs to delete. This can be a single blob, or multiple values can
             be supplied, where each value is either the name of the blob (str) or BlobProperties.
-        :type blobs: str or ~azure.storage.blob.BlobProperties
-        :param str delete_snapshots:
+        :type blobs: list[str] or list[~azure.storage.blob.BlobProperties]
+        :keyword str delete_snapshots:
             Required if a blob has associated snapshots. Values include:
              - "only": Deletes only the blobs snapshots.
              - "include": Deletes the blob along with all snapshots.
-        :param lease:
```

Comment:
If we are removing all these parameters - that's a breaking change?
Or were these never supported and these are errors in the documentation?

Reply:
no it's not, lease if for a specific blob, it doesn't make sense all blobs share the same lease right?

Reply:
It's not just lease though - we're removing all these documented parameters.
People might be passing these in - what was our SDK doing with them? We can't change that behaviour.

Reply:
the SDK was applying the same provided parameters(lease, etag, match_condition) to every blob.

Reply:
What if the lease was a container lease ID? Would that work?

Reply:
to delete a blob under a leased container, you don't need to provide a lease id 😳

Reply:
Okay :)